### PR TITLE
Add rapidfuzz/cci.20210517

### PR DESCRIPTION
**Pull request by hnOsmium0001**
Specify library name and version:  **rapidfuzz/cci.20210517**

https://github.com/maxbachmann/rapidfuzz-cpp
I omitted the -cpp suffix because I don't think we will be packaging the python version.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/rapidfuzz/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, tools
+import glob
+
+
```
Comment (SpaceIm) [inline recipes/rapidfuzz/all/conanfile.py:4]:
```suggestion

required_conan_version = ">=1.33.0"


```

File: recipes/rapidfuzz/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob(self.name + "-*/")[0]
+        tools.rename(extracted_dir, self._source_subfolder)
```
Comment (SpaceIm) [inline recipes/rapidfuzz/all/conanfile.py:21]:
```suggestion
        tools.get(**self.conan_data["sources"][self.version],
                  destination=self._source_subfolder, strip_root=True)
```

**Review by SpaceIm — DISMISSED** [review]

File: recipes/rapidfuzz/all/conandata.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20210513":
+    url: "https://github.com/maxbachmann/rapidfuzz-cpp/archive/d1e82379395cafc6d439c1c1e2cbe7512eaf2518.zip"
```
Comment (prince-chrismc) [inline recipes/rapidfuzz/all/conandata.yml:3]: Tar.gz?

**Review by prince-chrismc — DISMISSED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Comment by maxbachmann** [comment]
Thanks @hnOsmium0001 still had this on my todo list https://github.com/maxbachmann/rapidfuzz-cpp/issues/5

**Pull request merged**